ui/pages/desktop-error/render-desktop-error.js

```diff
@@ -61,10 +74,57 @@ export function renderDesktopError({
     );
   };
 
+  const getEventNameById = (id) => {
```

Comment:
none of these events follow our naming conventions defined here: https://www.notion.so/Naming-Conventions-95bf27517033451888053f7ab654d327